### PR TITLE
Mention embedded etcd as option in main install doc

### PR DESCRIPTION
**Pull request by cvlc**

### CODE REVIEW

**Comment by vfiftyfive** [comment]
Happy with the PR. Need @mdeneva to review the style. Also, MD linter failed, so it needs correction. I saw some warnings but doesn't seem to say why it failed.

**Comment by cvlc** [comment]
Thanks, @vfiftyfive, good spot on the linter errors - it was some trailing spaces, I've run markdownlint2-fix to resolve!

**Comment by hubvu** [comment]
- +1 from me on this PR.

- On a slightly related note - We should definitely discuss about how to present this page in the future as it seems like there is unnecessary duplication for installing Ondat on different versions of Kubernetes (ie 💡 "Ondat supports the five most recent Kubernetes releases, at minimum" should be sufficient and the list of supported versions should fall under a table in > https://docs.ondat.io/docs/prerequisites/)

**Review by mdeneva — APPROVED** [review]

File: docs/install/kubernetes.md [hunk 1]
```diff
@@ -35,7 +35,21 @@ curl -sSLo kubectl-storageos.tar.gz \
 > 💡 You can find binaries for different architectures and systems in [kubectl
 > plugin](https://github.com/storageos/kubectl-storageos/releases).
 
-### Install Ondat
+### Option A: Install Ondat (embedded etcd)
```
Comment (Arau) [inline docs/install/kubernetes.md:38]: Embedded is a mode we used to have in v1. But it is not the same thing as we are expecting as pods. There is docs about it and some customers could get confused. It is unfortunate, but I think we should find an alternative to the word embedded. May be etcd as pods, etcd operator, etcd in k8s, or something like that.

File: docs/install/kubernetes.md [hunk 2]
```diff
@@ -35,7 +35,21 @@ curl -sSLo kubectl-storageos.tar.gz \
...
+
+```bash
+kubectl storageos install \
+    --include-etcd \
+    --etcd-tls-enabled \
+    --admin-username "myuser" \
+    --admin-password "my-password"
+```
+
+> 💡 This is the easiest way to get up and running with Ondat, though
```
Comment (Arau) [inline docs/install/kubernetes.md:48]: Given that the production recommendation is still "bring-your-own etcd", I would put that as an Option A?

**Review by Arau — CHANGES_REQUESTED** [review]

File: docs/install/kubernetes.md [hunk 3]
```diff
@@ -94,7 +108,21 @@ curl -sSLo kubectl-storageos.tar.gz \
...
 
-### Install Ondat
+### Option A: Install Ondat (embedded etcd)
+
+```bash
+kubectl storageos install \
+    --include-etcd \
+    --etcd-tls-enabled \
+    --admin-username "myuser" \
+    --admin-password "my-password"
```
Comment (vfiftyfive) [inline docs/install/kubernetes.md:118]: Agree with Ferran, probably better to add the storageclass to use here and in the other references

**Pull request merged**